sci-physics/samurai/samurai-2.1.1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit fortran-2 autotools
+
+DESCRIPTION="Scattering AMplitudes from Unitarity-based Reduction Algorithm at the Integrand level."
+HOMEPAGE="https://samurai.hepforge.org/"
+SRC_URI="https://samurai.hepforge.org/downloads?f=${P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+qcdloop looptools"
+
+RDEPEND="
+	sci-physics/oneloop[dpkind,qpkind16,-qpkind]
+	qcdloop? ( sci-physics/qcdloop )
+	looptools? ( sci-physics/looptools )
+"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	# Fix that qcdloop and oneloop are already installed
+	sed -i 's/lib_LTLIBRARIES.*/lib_LTLIBRARIES = libsamurai.la/g' Makefile.am || die
+	local myeconfargs=(
+		FCFLAGS="${FCFLAGS} -std=legacy -fPIC -I${ESYSROOT}/usr/include"
+		--with-avh_olo="${ESYSROOT}"/usr/$(get_libdir)/libavh_olo.so
+		--with-avh_olo_precision=double
+		--with-precision=double
+		$(use_with qcdloop qcdloop "${ESYSROOT}"/usr)
```

Comment:
Does this need quotes again in case `ESYSROOT` has a space? I.e. https://github.com/gentoo/gentoo/blob/f84b09f5dd30d2e41d33afd6c284cc99cd9e22fa/app-admin/sudo/sudo-1.9.16_p2-r1.ebuild#L184 quotes twice.

Reply:
Yes it needs (double) quotes, the inside quotes ensure that the `ESYSROOT/usr` is one argument for `use_with`, the outside quotes ensures that the `ESYSROOT/usr` is one single element of the array.